docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 version: '3'
 services:
-  sesar:
+  sesar-rest-api:
     image: earthchemdocker/sesar-rest-api:1.0.0
     container_name: sesar-rest-api-1.0.0
     ports:
-     - "8080:8080"
+     - "8080:80"
```

Comment:
Wondering again if we should change this or live with 8080. We did this for consistency right? Does it really help?

Reply:
Right we should map it to 80 for consistency, changing it to port 80.